sift/client.py

```diff
@@ -12,7 +12,6 @@
 from . import version
 
 API_URL = 'https://api.siftscience.com'
-logging.basicConfig()
 sift_logger = logging.getLogger('sift_client')
```

Comment:
This logger will fail is logging hasn't been initialized previously, so it will work for you but not for someone who doesn't initialize the logger elsewhere.

After the creation of sift_logger, check if sift_logger has any handlers, and if not, run the logging.basicConfig command.

Should look like this
```
sift_logger = logging.getLogger('sift_client')
if not sift_logger.handlers:
  logging.basicConfig()
```